src/components/LoginModal/LoginModal.js

```diff
@@ -116,6 +172,15 @@ LoginModal.propTypes = {
 const mapStateToProps = (state) => ({
   modalForm: state.modalDisplay.modalForm,
   showModal: state.modalDisplay.showModal,
+  isLogged: state.isLogged.isLogged,
```

Comment:
Perhaps a more descriptive name here so that there is no confusion why there is two nested isLogged. Look into destructuring. It'll help make you be more descriptive.